package.json

```diff
@@ -22,14 +22,15 @@
   ],
   "scripts": {
     "preprocess": "node tasks/preprocess.js",
+    "precommit": "lint-staged",
     "bundle": "node tasks/bundle.js",
     "header": "node tasks/header.js",
     "stats": "node tasks/stats.js",
     "build": "npm run preprocess && npm run bundle && npm run header && npm run stats",
     "cibuild": "npm run preprocess && node tasks/cibundle.js",
     "watch": "node tasks/watch.js",
-    "lint": "eslint --version && eslint .",
```

Comment:
is `prettier` able to detect unused variables and variable used out of scope?

If not, I think it's better to keep using `eslint` for linting.